client/jetpack-cloud/sections/partner-portal/utils.ts

```diff
@@ -192,3 +192,10 @@ export function selectAlphaticallySortedProductOptions(
 ): APIProductFamilyProduct[] {
 	return sortBy( selectProductOptions( families ), ( product ) => product.name );
 }
+
+export function isJetpackBundle( product: APIProductFamilyProduct | string ) {
+	if ( typeof product === 'string' ) {
+		return [ 'jetpack-complete', 'jetpack-security-t1', 'jetpack-security-t2' ].includes( product );
+	}
```

Comment:
How about moving these bundles into a separate array type that we can export? I think this could be beneficial if we have similar functions.